admin-dev/themes/new-theme/.webpack/dev.js

```diff
@@ -11,10 +15,62 @@ function devConfig() {
     {
       devtool: 'inline-source-map',
       devServer: {
+        client: {
+          logging: 'error',
+          progress: false,
+          overlay: {
+            errors: true,
+            warnings: false,
+          },
+        },
         hot: true,
-        contentBase: path.resolve(__dirname, '/../public'),
-        publicPath: '/',
+        static: {
+          directory: path.join(__dirname, '/../public'),
+          watch: false,
+        },
+        port: 3000,
+        watchFiles: [
+          path.join(__dirname, '/../**/*.tpl'),
+          path.join(__dirname, '../js/**/*.js'),
+          path.join(__dirname, '../scss/**/*.scss'),
+        ],
+        open: true,
+        proxy: {
+          '**': {
+            target: process.env.PS_URL,
+            secure: false,
+            changeOrigin: true,
+          },
+        },
+        devMiddleware: {
+          publicPath: path.join(__dirname, '/../public'),
+          writeToDisk: (filePath) => !(/hot-update/.test(filePath)),
```

Comment:
hot-update should be in the gitignore no?

Reply:
every assets are ignored, it doesn't appear inside my git modified files so I didn't ask myself about this